Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -643,6 +680,21 @@ function QuestieOptions.tabs.tracker:Initialize()
                     QuestieTracker:Update()
                 end
             },
+            fontOutline = {
+                type = "select",
+                dialogControl = 'LSM30_Font',
+                order = 3.85,
+                values = outlineTable,
+                style = 'dropdown',
+                name = function() return l10n('Outline for Zones, Titles, and Objectives'); end,
+                desc = function() return l10n('The outline used for Quest Zones, Titles, and Objectives in Tracker.'); end,
```

Comment:
Could you add the translations (or the emtpy entries) to https://github.com/Questie/Questie/blob/master/Localization/Translations/Options/Tracker.lua as well?

Reply:
Fixed!